src/sidebar/components/menu-item.js

```diff
@@ -45,7 +63,9 @@ export default function MenuItem({
   const hasLeftIcon = icon || isSubmenuItem;
   const hasRightIcon = icon && isSubmenuItem;
 
-  const menuItemRef = useRef(null);
+  const menuItemRef = useRef(
+    /** @type {(HTMLAnchorElement&HTMLDivElement)|null} */ (null)
```

Comment:
```suggestion
    /** @type {(HTMLAnchorElement & HTMLDivElement)|null} */ (null)
```

This is just a personal preference and we don't have an agreed convention here, but whereas `X|Y` is easy to read without spaces, I find `X & Y` to be easier to read with spaces.

Reply:
I like your suggestion, lets go with spaces.